src/fromager/commands/build.py

```diff
@@ -141,6 +141,19 @@ def _build(
     req: Requirement,
     source_download_url: str,
 ) -> pathlib.Path:
+    pbi = wkctx.package_build_info(req)
+    build_tag_from_settings = pbi.build_tag(resolved_version)
+    build_tag = build_tag_from_settings if build_tag_from_settings else (0, "")
+    build_tag_as_string = " ".join(map(str, build_tag))
```

Comment:
This will put a space in the filename, right? That tends to make shell scripting harder, so let's use another separator. Could we use the same standard that's used to add a build tag to a version in the wheel filename?

Reply:
oh yes okay. I replaced it with the same standard used to add a build tag to a version in the wheel filename